src/overlays/actors/ovl_En_GeldB/z_en_geldb.h

```diff
@@ -6,14 +6,42 @@
 
 struct EnGeldB;
 
+typedef enum {
+    /* 00 */ GELDB_LIMB_NONE,
+    /* 01 */ GELDB_LIMB_ROOT,
+    /* 02 */ GELDB_LIMB_TORSO,
+    /* 03 */ GELDB_LIMB_NECK,
+    /* 04 */ GELDB_LIMB_PONYTAIL,
+    /* 05 */ GELDB_LIMB_VEIL,
+    /* 06 */ GELDB_LIMB_HEAD,
+    /* 07 */ GELDB_LIMB_UPPER_ARM_R,
+    /* 08 */ GELDB_LIMB_FOREARM_R,
+    /* 09 */ GELDB_LIMB_WRIST_R,
+    /* 10 */ GELDB_LIMB_HAND_R,
+    /* 11 */ GELDB_LIMB_SWORD_R,
+    /* 12 */ GELDB_LIMB_UPPER_ARM_L,
+    /* 13 */ GELDB_LIMB_FOREARM_L,
+    /* 14 */ GELDB_LIMB_WRIST_L,
+    /* 15 */ GELDB_LIMB_HAND_L,
+    /* 16 */ GELDB_LIMB_SWORD_L,
+    /* 17 */ GELDB_LIMB_THIGH_L,
+    /* 18 */ GELDB_LIMB_SHIN_L,
+    /* 19 */ GELDB_LIMB_FOOT_L,
+    /* 20 */ GELDB_LIMB_THIGH_R,
+    /* 21 */ GELDB_LIMB_SHIN_R,
+    /* 22 */ GELDB_LIMB_FOOT_R,
```

Comment:
Pretty minor but in player I went with `_R_FOOT` and such instead of `_FOOT_R`, and we should probably be consistent in the naming for these

Reply:
Any preference? I think direction on the end makes sense cause its more of like a single letter suffix. but i dont mind too much either way

Reply:
They seem grouped by direction so I think treating the direction as a prefix makes slightly more sense, but that's about it

Reply:
fair enough, will change